symbolic-symcache/tests/test_writer.rs

```diff
@@ -215,3 +217,54 @@ fn test_trailing_marker() -> Result<(), Error> {
 
     Ok(())
 }
+
+// FIXME: This is a huge pain, can't this be simpler somehow?
+struct OwnedBcSymbolMap(SelfCell<ByteView<'static>, BcSymbolMap<'static>>);
+
+impl Transformer for OwnedBcSymbolMap {
+    fn transform_function<'f>(&'f self, f: transform::Function<'f>) -> transform::Function<'f> {
+        self.0.get().transform_function(f)
```

Comment:
i'm being blind, but where is this actual implementation living?

Reply:
Its the one that actually needs the `map_cow` machinery